diff --git a/server/hooks/gen.go b/server/hooks/gen.go
index ac23515..6a861c8 100644
--- a/server/hooks/gen.go
+++ b/server/hooks/gen.go
@@ -41,13 +41,7 @@ func GenerateHooks(_ context.Context, cfg *config.Config, repo string) error {
 		return err
 	}
 
-	ex, err := os.Executable()
-	if err != nil {
-		return err
-	}
-
-	// Convert to forward slashes for Windows.
-	ex = filepath.ToSlash(ex)
+	ex := "soft"
 
 	for _, hook := range []string{
 		PreReceiveHook,
@@ -94,7 +88,7 @@ func GenerateHooks(_ context.Context, cfg *config.Config, repo string) error {
 
 		// Write the soft-serve hook inside ${hook}.d directory.
 		hp = filepath.Join(hp, "soft-serve")
-		err = os.WriteFile(hp, data.Bytes(), os.ModePerm) //nolint:gosec
+		err := os.WriteFile(hp, data.Bytes(), os.ModePerm) //nolint:gosec
 		if err != nil {
 			log.WithPrefix("hooks").Error("failed to write hook", "err", err)
 			continue
