diff --git a/catppuccin-frappe.tmuxtheme b/catppuccin-frappe.tmuxtheme
index 6c3fc7b..9c68d25 100755
--- a/catppuccin-frappe.tmuxtheme
+++ b/catppuccin-frappe.tmuxtheme
@@ -56,11 +56,11 @@ setw window-status-style "fg=${thm_fg},bg=${thm_bg},none"
 # --------=== Statusline
 
 set status-left ""
-set status-right "#[fg=$thm_pink,bg=$thm_bg,nobold,nounderscore,noitalics]#[fg=$thm_bg,bg=$thm_pink,nobold,nounderscore,noitalics] #[fg=$thm_fg,bg=$thm_gray] #W #{?client_prefix,#[fg=$thm_red],#[fg=$thm_green]}#[bg=$thm_gray]#{?client_prefix,#[bg=$thm_red],#[bg=$thm_green]}#[fg=$thm_bg] #[fg=$thm_fg,bg=$thm_gray] #S "
+set status-right "#[fg=$thm_yellow,bg=$thm_bg,nobold,nounderscore,noitalics]#[fg=$thm_bg,bg=$thm_yellow,nobold,nounderscore,noitalics] #[fg=$thm_fg,bg=$thm_gray] #{USER}@#H #{?client_prefix,#[fg=$thm_red],#[fg=$thm_green]}#[bg=$thm_gray]#{?client_prefix,#[bg=$thm_red],#[bg=$thm_green]}#[fg=$thm_bg] #[fg=$thm_fg,bg=$thm_gray] %T [%Z] "
 
 # current_dir
-setw window-status-format "#[fg=$thm_bg,bg=$thm_blue] #I #[fg=$thm_fg,bg=$thm_gray] #{b:pane_current_path} "
-setw window-status-current-format "#[fg=$thm_bg,bg=$thm_orange] #I #[fg=$thm_fg,bg=$thm_bg] #{b:pane_current_path} "
+setw window-status-format "#[fg=$thm_bg,bg=$thm_blue] #I #[fg=$thm_black4,bg=$thm_bg] [#W] "
+setw window-status-current-format "#[fg=$thm_bg,bg=$thm_orange] #I#{?window_zoomed_flag,*,} #[fg=$thm_fg,bg=$thm_bg] #[bold][#W]#[nobold]@#[underscore]#{b:pane_current_path}#[nounderscore] "
 
 # parent_dir/current_dir
 # setw window-status-format "#[fg=colour232,bg=colour111] #I #[fg=colour222,bg=colour235] #(echo '#{pane_current_path}' | rev | cut -d'/' -f-2 | rev) "
diff --git a/catppuccin-latte.tmuxtheme b/catppuccin-latte.tmuxtheme
index 1ddb188..be2aeea 100755
--- a/catppuccin-latte.tmuxtheme
+++ b/catppuccin-latte.tmuxtheme
@@ -56,11 +56,11 @@ setw window-status-style "fg=${thm_fg},bg=${thm_bg},none"
 # --------=== Statusline
 
 set status-left ""
-set status-right "#[fg=$thm_pink,bg=$thm_bg,nobold,nounderscore,noitalics]#[fg=$thm_bg,bg=$thm_pink,nobold,nounderscore,noitalics] #[fg=$thm_fg,bg=$thm_gray] #W #{?client_prefix,#[fg=$thm_red],#[fg=$thm_green]}#[bg=$thm_gray]#{?client_prefix,#[bg=$thm_red],#[bg=$thm_green]}#[fg=$thm_bg] #[fg=$thm_fg,bg=$thm_gray] #S "
+set status-right "#[fg=$thm_yellow,bg=$thm_bg,nobold,nounderscore,noitalics]#[fg=$thm_bg,bg=$thm_yellow,nobold,nounderscore,noitalics] #{USER}@#H #{?client_prefix,#[fg=$thm_red],#[fg=$thm_green]}#[bg=$thm_gray]#{?client_prefix,#[bg=$thm_red],#[bg=$thm_green]}#[fg=$thm_bg] #[fg=$thm_fg,bg=$thm_gray] %T [%Z] "
 
 # current_dir
-setw window-status-format "#[fg=$thm_bg,bg=$thm_blue] #I #[fg=$thm_fg,bg=$thm_gray] #{b:pane_current_path} "
-setw window-status-current-format "#[fg=$thm_bg,bg=$thm_orange] #I #[fg=$thm_fg,bg=$thm_bg] #{b:pane_current_path} "
+setw window-status-format "#[fg=$thm_bg,bg=$thm_blue] #I #[fg=$thm_black4,bg=$thm_bg] [#W] "
+setw window-status-current-format "#[fg=$thm_bg,bg=$thm_orange] #I#{?window_zoomed_flag,*,} #[fg=$thm_fg,bg=$thm_bg] #[bold][#W]#[nobold]@#[underscore]#{b:pane_current_path}#[nounderscore] "
 
 # parent_dir/current_dir
 # setw window-status-format "#[fg=colour232,bg=colour111] #I #[fg=colour222,bg=colour235] #(echo '#{pane_current_path}' | rev | cut -d'/' -f-2 | rev) "
diff --git a/catppuccin-macchiato.tmuxtheme b/catppuccin-macchiato.tmuxtheme
index cdaaa89..feb1e8a 100755
--- a/catppuccin-macchiato.tmuxtheme
+++ b/catppuccin-macchiato.tmuxtheme
@@ -56,11 +56,11 @@ setw window-status-style "fg=${thm_fg},bg=${thm_bg},none"
 # --------=== Statusline
 
 set status-left ""
-set status-right "#[fg=$thm_pink,bg=$thm_bg,nobold,nounderscore,noitalics]#[fg=$thm_bg,bg=$thm_pink,nobold,nounderscore,noitalics] #[fg=$thm_fg,bg=$thm_gray] #W #{?client_prefix,#[fg=$thm_red],#[fg=$thm_green]}#[bg=$thm_gray]#{?client_prefix,#[bg=$thm_red],#[bg=$thm_green]}#[fg=$thm_bg] #[fg=$thm_fg,bg=$thm_gray] #S "
+set status-right "#[fg=$thm_yellow,bg=$thm_bg,nobold,nounderscore,noitalics]#[fg=$thm_bg,bg=$thm_yellow,nobold,nounderscore,noitalics] #{USER}@#H #{?client_prefix,#[fg=$thm_red],#[fg=$thm_green]}#[bg=$thm_gray]#{?client_prefix,#[bg=$thm_red],#[bg=$thm_green]}#[fg=$thm_bg] #[fg=$thm_fg,bg=$thm_gray] %T [%Z] "
 
 # current_dir
-setw window-status-format "#[fg=$thm_bg,bg=$thm_blue] #I #[fg=$thm_fg,bg=$thm_gray] #{b:pane_current_path} "
-setw window-status-current-format "#[fg=$thm_bg,bg=$thm_orange] #I #[fg=$thm_fg,bg=$thm_bg] #{b:pane_current_path} "
+setw window-status-format "#[fg=$thm_bg,bg=$thm_blue] #I #[fg=$thm_black4,bg=$thm_bg] [#W] "
+setw window-status-current-format "#[fg=$thm_bg,bg=$thm_orange] #I#{?window_zoomed_flag,*,} #[fg=$thm_fg,bg=$thm_bg] #[bold][#W]#[nobold]@#[underscore]#{b:pane_current_path}#[nounderscore] "
 
 # parent_dir/current_dir
 # setw window-status-format "#[fg=colour232,bg=colour111] #I #[fg=colour222,bg=colour235] #(echo '#{pane_current_path}' | rev | cut -d'/' -f-2 | rev) "
diff --git a/catppuccin-mocha.tmuxtheme b/catppuccin-mocha.tmuxtheme
index e8178ec..869779d 100755
--- a/catppuccin-mocha.tmuxtheme
+++ b/catppuccin-mocha.tmuxtheme
@@ -56,11 +56,11 @@ setw window-status-style "fg=${thm_fg},bg=${thm_bg},none"
 # --------=== Statusline
 
 set status-left ""
-set status-right "#[fg=$thm_pink,bg=$thm_bg,nobold,nounderscore,noitalics]#[fg=$thm_bg,bg=$thm_pink,nobold,nounderscore,noitalics] #[fg=$thm_fg,bg=$thm_gray] #W #{?client_prefix,#[fg=$thm_red],#[fg=$thm_green]}#[bg=$thm_gray]#{?client_prefix,#[bg=$thm_red],#[bg=$thm_green]}#[fg=$thm_bg] #[fg=$thm_fg,bg=$thm_gray] #S "
+set status-right "#[fg=$thm_yellow,bg=$thm_bg,nobold,nounderscore,noitalics]#[fg=$thm_bg,bg=$thm_yellow,nobold,nounderscore,noitalics] #{USER}@#H #{?client_prefix,#[fg=$thm_red],#[fg=$thm_green]}#[bg=$thm_yellow]#{?client_prefix,#[bg=$thm_red],#[bg=$thm_green]}#[fg=$thm_bg] %T [%Z] "
 
 # current_dir
-setw window-status-format "#[fg=$thm_bg,bg=$thm_blue] #I #[fg=$thm_fg,bg=$thm_gray] #{b:pane_current_path} "
-setw window-status-current-format "#[fg=$thm_bg,bg=$thm_orange] #I #[fg=$thm_fg,bg=$thm_bg] #{b:pane_current_path} "
+setw window-status-format "#[fg=$thm_bg,bg=$thm_blue] #I #[fg=$thm_black4,bg=$thm_bg] [#W] "
+setw window-status-current-format "#[fg=$thm_bg,bg=$thm_orange] #I#{?window_zoomed_flag,*,} #[fg=$thm_fg,bg=$thm_bg] #[bold][#W]#[nobold]@#[underscore]#{b:pane_current_path}#[nounderscore] "
 
 # parent_dir/current_dir
 # setw window-status-format "#[fg=colour232,bg=colour111] #I #[fg=colour222,bg=colour235] #(echo '#{pane_current_path}' | rev | cut -d'/' -f-2 | rev) "
