diff --git a/catppuccin.tmux b/catppuccin.tmux
index 47d06e9..aded9a1 100755
--- a/catppuccin.tmux
+++ b/catppuccin.tmux
@@ -68,10 +68,13 @@ main() {
   readonly show_directory="#[fg=$thm_pink,bg=$thm_bg,nobold,nounderscore,noitalics]#[fg=$thm_bg,bg=$thm_pink,nobold,nounderscore,noitalics]  #[fg=$thm_fg,bg=$thm_gray] #{b:pane_current_path} #{?client_prefix,#[fg=$thm_red]"
   readonly show_window="#[fg=$thm_pink,bg=$thm_bg,nobold,nounderscore,noitalics]#[fg=$thm_bg,bg=$thm_pink,nobold,nounderscore,noitalics] #[fg=$thm_fg,bg=$thm_gray] #W #{?client_prefix,#[fg=$thm_red]"
   readonly show_session="#[fg=$thm_green]}#[bg=$thm_gray]#{?client_prefix,#[bg=$thm_red],#[bg=$thm_green]}#[fg=$thm_bg] #[fg=$thm_fg,bg=$thm_gray] #S "
-  readonly show_directory_in_window_status="#[fg=$thm_bg,bg=$thm_blue] #I #[fg=$thm_fg,bg=$thm_gray] #{b:pane_current_path} "
-  readonly show_directory_in_window_status_current="#[fg=$thm_bg,bg=$thm_orange] #I #[fg=$thm_fg,bg=$thm_bg] #{b:pane_current_path} "
+  readonly show_directory_in_window_status="#[fg=$thm_bg,bg=$thm_blue] #I #[fg=$thm_black4,bg=$thm_bg] [#W] "
+  readonly show_directory_in_window_status_current="#[fg=$thm_bg,bg=$thm_orange] #I#{?window_zoomed_flag,*,} #[fg=$thm_fg,bg=$thm_bg] #[fg=$thm_black4][#[fg=$thm_fg,bold]#W#[fg=$thm_black4,nobold]]#
+       [fg=$thm_fg]@#[underscore]#{b:pane_current_path}#[nounderscore] "
   readonly show_window_in_window_status="#[fg=$thm_fg,bg=$thm_bg] #W #[fg=$thm_bg,bg=$thm_blue] #I#[fg=$thm_blue,bg=$thm_bg]#[fg=$thm_fg,bg=$thm_bg,nobold,nounderscore,noitalics] "
   readonly show_window_in_window_status_current="#[fg=$thm_fg,bg=$thm_gray] #W #[fg=$thm_bg,bg=$thm_orange] #I#[fg=$thm_orange,bg=$thm_bg]#[fg=$thm_fg,bg=$thm_bg,nobold,nounderscore,noitalics] "
+  readonly show_user_at_hostname="#[fg=$thm_yellow,bg=$thm_bg,nobold,nounderscore,noitalics]#[fg=$thm_bg,bg=$thm_yellow,nobold,nounderscore,noitalics] #[fg=$thm_fg,bg=$thm_gray] #{USER}@#H "
+  readonly show_clock_with_tz="#{?client_prefix,#[fg=$thm_red],#[fg=$thm_green]}#[bg=$thm_gray]#{?client_prefix,#[bg=$thm_red],#[bg=$thm_green]}#[fg=$thm_bg] #[fg=$thm_fg,bg=$thm_gray] %T [%Z] "
 
   # Right column 1 by default shows the Window name.
   local right_column1=$show_window
@@ -94,7 +97,7 @@ main() {
 
   set status-left ""
 
-  set status-right "${right_column1},${right_column2}"
+  set status-right "${show_user_at_hostname}${show_clock_with_tz}"
 
   setw window-status-format "${window_status_format}"
   setw window-status-current-format "${window_status_current_format}"
-- 
2.38.1

