diff --git a/plugin/wakatime.vim b/plugin/wakatime.vim
index 375e668..04b09f8 100644
--- a/plugin/wakatime.vim
+++ b/plugin/wakatime.vim
@@ -680,8 +680,8 @@ EOF
             let api_key = s:GetIniSetting('settings', 'apikey')
         endif
 
-        let api_key = inputsecret("[WakaTime] Enter your wakatime.com api key: ", api_key)
-        call s:SetIniSetting('settings', 'api_key', api_key)
+        " let api_key = inputsecret("[WakaTime] Enter your wakatime.com api key: ", api_key)
+        " call s:SetIniSetting('settings', 'api_key', api_key)
     endfunction
 
     function! s:EnableDebugMode()
