From 1cf3f0bb5f71227c649ce6fc7547de5447115717 Mon Sep 17 00:00:00 2001
From: Gaoyang Zhang <gy@blurgy.xyz>
Date: Thu, 27 Oct 2022 14:43:19 +0800
Subject: [PATCH] disable interactive secret prompt

Signed-off-by: Gaoyang Zhang <gy@blurgy.xyz>
---
 plugin/wakatime.vim | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/plugin/wakatime.vim b/plugin/wakatime.vim
index 375e668..32b1d4e 100644
--- a/plugin/wakatime.vim
+++ b/plugin/wakatime.vim
@@ -679,9 +679,6 @@ EOF
         if api_key == ''
             let api_key = s:GetIniSetting('settings', 'apikey')
         endif
-
-        let api_key = inputsecret("[WakaTime] Enter your wakatime.com api key: ", api_key)
-        call s:SetIniSetting('settings', 'api_key', api_key)
     endfunction
 
     function! s:EnableDebugMode()
-- 
2.38.0

